publish-release.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Tag and publish a new release or `purescm` to npmjs.org
+#
+# Usage:
+#
+# ./publish-release.sh <major|minor|patch...>
+#
+
+set -eu
+
+if [ $# -eq 0 ]
+  then
+    echo "No arguments supplied"
+    exit 1
+fi
+
+# bump npm version and tag commit
+npm version $1
+
+# bump also in spago
+git add spago.yaml
+git commit --amend --no-edit
```

Comment:
Why the `--amend --no-edit`?

Reply:
I wanted to amend the commit that `npm version` makes. I could just make another commit, or I can prevent `npm version` not do the commit, but then I need to also do a `git tag` here (which `npm version` does automatically).